openshift/kubecon-demo.sh

```diff
@@ -29,12 +28,11 @@ function run_demo(){
   curl -H "Host: helloworld-openshift.myproject.example.com" "http://${ip}:${port}/health" || return 1
```

Comment:
Can you please remove the port from this call and also remove the the `port` variable definition? After long debugging I'm still not sure why it happens but it seems that removing the port makes it work. It's probably an issue with the OpenShift cluster that prevents connecting to the node port. Strange thing is that yesterday it was working :-/